lib/install/dispatcher/dispatcher.go

```diff
@@ -89,6 +89,11 @@ type Event struct {
 	Status Status
 }
 
+// IsCompleted returns true if this status indicates completion
```

Comment:
Nit: doc doesn't seem to exactly match implementation. I assume CompletedPending has a slight semantic difference to completed (or it wouldn't exist), so I think the comment should reflect this semantic so it doesn't accidentally get miss-used?